### PR TITLE
[Doc Improvement][Update universal action description for Action.Execute]

### PR DESCRIPTION
**Pull request by v-ajiths**
Updated the Universal action description

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/task-modules-and-cards/cards/Universal-actions-for-adaptive-cards/Overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3c589132f71fb9bba7fcb6a36d17fe5c0d98111e/msteams-platform%2Ftask-modules-and-cards%2Fcards%2FUniversal-actions-for-adaptive-cards%2FOverview.md) | 93 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a80a6ea4-534b-4da7-a248-347e3daec608) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/task-modules-and-cards/cards/Universal-actions-for-adaptive-cards/Overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d280e483827d6315500765566ce73f73281b427e/msteams-platform%2Ftask-modules-and-cards%2Fcards%2FUniversal-actions-for-adaptive-cards%2FOverview.md) | 93 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1142e822-0cbc-4ce4-95ec-706fa4bae891) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/task-modules-and-cards/cards/Universal-actions-for-adaptive-cards/Overview.md [hunk 1]
```diff
@@ -36,7 +36,11 @@ The following image shows the current inconsistent action model:
 
 :::image type="content" source="~/assets/images/adaptive-cards/current-teams-outlook-action-model.png" alt-text="Inconsistent action model":::
 
-With the Universal Actions for Adaptive Cards, you can use `Action.Execute` for action handling across different platforms. `Action.Execute` works across hubs including Teams and Outlook. In addition, an Adaptive Card can be returned as response for an `Action.Execute` triggered invoke request.
+With the Universal Actions for Adaptive Cards, you can use `Action.Execute` for action handling across different platforms.
+
+`Action.Execute` works across hubs, including Teams and Outlook, but it isn't a replacement for `Action.Submit`. However, in a few cases, such as if you want some external system to do an action and the result of the action to be sent back to your conversation, which uses [Messaging Extension](../../../messaging-extensions/what-are-messaging-extensions.md), `Action.Execute` isn't supported. For [link unfurling cards](../../../messaging-extensions/how-to/link-unfurling.md) like Hero and Thumbnail Cards, you need to call `Action.Submit`.
```
Comment (surbhigupta12) [inline msteams-platform/task-modules-and-cards/cards/Universal-actions-for-adaptive-cards/Overview.md:41]: Too wordy. Please rewrite.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/task-modules-and-cards/cards/Universal-actions-for-adaptive-cards/Overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/cc944754df7b0ea2b105ba8b585749e7176616ec/msteams-platform%2Ftask-modules-and-cards%2Fcards%2FUniversal-actions-for-adaptive-cards%2FOverview.md) | 92 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c019c455-0944-4bf0-ba6c-832b569e29f6) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/task-modules-and-cards/cards/Universal-actions-for-adaptive-cards/Overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a38fa942e194ad5d3e5cc40c178191a181d6a398/msteams-platform%2Ftask-modules-and-cards%2Fcards%2FUniversal-actions-for-adaptive-cards%2FOverview.md) | 92 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9660d77b-5eac-4620-af73-9ee724b88702) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/task-modules-and-cards/cards/Universal-actions-for-adaptive-cards/Overview.md (hunk 1 above)
Comment (v-ajiths) [reply msteams-platform/task-modules-and-cards/cards/Universal-actions-for-adaptive-cards/Overview.md]: Updated as suggested.

**Review by v-habibaumme — DISMISSED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/task-modules-and-cards/cards/Universal-actions-for-adaptive-cards/Overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/7551ebf630155a48efe523d420ee5366f319fe79/msteams-platform%2Ftask-modules-and-cards%2Fcards%2FUniversal-actions-for-adaptive-cards%2FOverview.md) | 92 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/127e0415-c914-475e-a2f9-54810388ee93) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/task-modules-and-cards/cards/Universal-actions-for-adaptive-cards/Overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2382c6edb94c8576d6cd8707b2c892624c663237/msteams-platform%2Ftask-modules-and-cards%2Fcards%2FUniversal-actions-for-adaptive-cards%2FOverview.md) | 92 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1bc391b7-b132-4f6f-9f77-7af9e3e08ce9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/task-modules-and-cards/cards/Universal-actions-for-adaptive-cards/Overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6697b9f47b300c83f6c6d11dd5ffe0ae12a4ac35/msteams-platform%2Ftask-modules-and-cards%2Fcards%2FUniversal-actions-for-adaptive-cards%2FOverview.md) | 92 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ffd1d3be-1f27-4361-abd6-d43bc6908bef) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-habibaumme — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/task-modules-and-cards/cards/Universal-actions-for-adaptive-cards/Overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5624a0ca0ec92a8c84527cbaa3c0c511a97e6e6c/msteams-platform%2Ftask-modules-and-cards%2Fcards%2FUniversal-actions-for-adaptive-cards%2FOverview.md) | 92 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ff132f54-5320-46f7-89d7-b7c71bacfaf7) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/task-modules-and-cards/cards/Universal-actions-for-adaptive-cards/Overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8780015315bd9e21578e0c41e484b1d1c6ee5bd7/msteams-platform%2Ftask-modules-and-cards%2Fcards%2FUniversal-actions-for-adaptive-cards%2FOverview.md) | 92 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1645558f-149f-4c6e-a196-5bd4de8eae12) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/task-modules-and-cards/cards/Universal-actions-for-adaptive-cards/Overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8288448af2d4431df37712cd24c0a7a2486caa3a/msteams-platform%2Ftask-modules-and-cards%2Fcards%2FUniversal-actions-for-adaptive-cards%2FOverview.md) | 92 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b83e510b-0355-4995-9e66-cabdfcf9803e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by sayali-MSFT — APPROVED** [review]
Approved.

**Review by Wajeed-msft — APPROVED** [review]

**Review by v-habibaumme — APPROVED** [review]

**Comment by aosolis** [comment]
@surbhigupta12 was this reviewed by dev or PM? Eoin came across this for a partner scenario, and it wasn't clear to us what this means

> or example, if you want an external system to do an action and the result of the action must be sent back to your conversation using [Messaging Extension](https://learn.microsoft.com/en-us/microsoftteams/platform/messaging-extensions/what-are-messaging-extensions), Action.Execute isn't supported.

Also this doesn't quite make sens,e because cards don't "call" Action.Submit.

> For [link unfurling cards](https://learn.microsoft.com/en-us/microsoftteams/platform/messaging-extensions/how-to/link-unfurling) such as hero and thumbnail cards, you must call Action.Submit.

**Comment by sayali-MSFT** [comment]
@aosolis - As per document these are the scenario where we can use the Messaging extension. messaging Extension is not supported to Action.Execute. for clarification purpose add the example here.
1.https://learn.microsoft.com/en-us/microsoftteams/platform/messaging-extensions/what-are-messaging-extensions?tabs=dotnet#scenarios-where-message-extensions-are-used
2.https://learn.microsoft.com/en-us/microsoftteams/platform/messaging-extensions/how-to/link-unfurling?tabs=dotnet%2Cadvantages#handle-the-composeextensionquerylink-invoke

 Could you please confirm if this is what we need to update in this doc? Or is there anything else want to add?
